src/components/views/game/game/videochat/VideoChat.tsx

```diff
@@ -69,6 +69,17 @@ export const VideoChat = observer(() => {
                 iframe.style.width = "100%"
               }}
             />
+
+            <div> </div>
```

Comment:
These div should be unnecessary.
Spaces between components should be done with padding and margin.

Reply:
🙃 absolutely, I _would_ actually know that